src/test/ts/part2/Exercise2ArrayFunctionsTest.ts

```diff
@@ -8,26 +8,35 @@ describe('Exercise2ArrayFunctionsTest', () => {
   });
 
   it('frog names', () => {
-    assert.deepEqual(Ex.frogNames(Ex.myFrogs), [ 'frog1', 'frog2', 'loudfrog', 'quietfrog' ], 'frog names');
+    assert.deepEqual(Ex.frogNames(Ex.myFrogs), ['frog1', 'frog2', 'loudfrog', 'quietfrog'], 'frog names');
   });
 
   it('frog ages', () => {
     // TODO: write a test for your frog ages function
+    assert.deepEqual(Ex.frogAges(Ex.myFrogs), [3, 4, 1, 10], 'frog names');
```

Comment:
Not sure if it really matters here since this is a exercise on using the libraries, test tools and code style but `names` should be `ages` here.